### ISSUE TITLE
[GFX] When a Cup starts show a manialink/image like in TM2020 Ranked

### ISSUE THREAD

**Issue by w1lla** [skybaks/pyplanet-cup_manager#47]
Should give it a better feeling for players. GUI wise in teams for instance like Match/Cup Starting in 5,4,3,2,1 where it counts down. That should give notice to players.

**Comment by w1lla** [comment]
![mp_rework_GFX](https://user-images.githubusercontent.com/1784761/190928863-8469043c-c421-4ee8-8ded-c639bb798b47.png)

**Comment by skybaks** [comment]
Yeah i think something like this could be super cool. I wonder, tho, if it might be more appropriate for it live inside the mode script instead of the controller.

It would be catastrophic if we ever got into a situation where the screen didnt clear in the appropriate time for all the players on the server. That idea makes me very nervous about adding this feature

**Comment by w1lla** [comment]
Well its only active in Teams_Matchmaking gamemode, the old Teams Gamemode doesn't show this. So it could be a feature request eventually for Nadeo to make it that way possible.
